include/ccglib/common/value_type.h

```diff
@@ -11,12 +11,21 @@
 
 namespace ccglib {
 
-enum ValueType { int1, int32, bfloat16, float8e4m3, float16, float32 };
+enum ValueType {
+  int1,
+  int32,
+  bfloat16,
+  float8e5m2,
+  float8e4m3,
+  float16,
+  float32
+};
```

Comment:
The order is already somewhat arbitrary, but I think `e4m3` should definitely go before `e5m2`:
```suggestion
enum ValueType {
  int1,
  int32,
  bfloat16,
  float8e4m3,
  float8e5m2,
  float16,
  float32
};
```